src/core/symbol/SymbolIdGenerator.ts

```diff
@@ -90,4 +121,13 @@ export class SymbolIdGenerator {
     public static isValidNamespaceName(name: string): boolean {
         return NamespaceConst.name_pattern.test(name);
     }
+
+    /**
+     * It encodes a namespace id as an unresolved alias to be used when serializing unresolved addresses with catbuffer.
+     * @param networkType - the network type
+     * @param namespaceId - the network id as bigint.
+     */
+    public static encodeUnresolvedAddress(networkType: number, namespaceId: bigint): Uint8Array {
```

Comment:
- put it in Address class perhaps?
- networkType => network: Network? or perhaps rename to networkIdentifier to be consistent

Reply:
Is `networkIdentifier` the right name? if in the future we have 2 testnets, the `0x98 / 152` would be used to identify both networks? @gimer @Jaguar0625